packages/web-frontend/src/components/Marker/markerColors.js

```diff
@@ -24,9 +26,22 @@ const SCALE_TYPE_TO_COLOR_SCHEME = {
   [SCALE_TYPES.PERFORMANCE]: PERFORMANCE_COLOR_SCHEME,
   [SCALE_TYPES.PERFORMANCE_DESC]: PERFORMANCE_COLOR_SCHEME,
   [SCALE_TYPES.NEUTRAL]: DEFAULT_COLOR_SCHEME,
+  [SCALE_TYPES.NEUTRAL_SWAP]: SWAP_DEFAULT_COLOR_SCHEME,
   [SCALE_TYPES.TIME]: TIME_COLOR_SCHEME,
 };
 
+const rgbSet = [
```

Comment:
This seems to be specific to the to the heatmap? Maybe `HEATMAP_DEFAULT_RGB_SET`?

Reply:
More specific👍